From 924d3ee1925dcbe0fd4541282b053e855984b1f8 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Sat, 29 Dec 2018 16:35:38 +0800
Subject: [PATCH] dt-bindings: display: rockchip: lvds: add support for generic
 phy

This patch adds support for generic phy as an optional.

Change-Id: Idcb30be4a88a688be839b45a96d3b78258d63a7d
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 .../devicetree/bindings/display/rockchip/rockchip-lvds.txt    | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/rockchip/rockchip-lvds.txt b/Documentation/devicetree/bindings/display/rockchip/rockchip-lvds.txt
index b686ae7d24c9..247531edacc1 100644
--- a/Documentation/devicetree/bindings/display/rockchip/rockchip-lvds.txt
+++ b/Documentation/devicetree/bindings/display/rockchip/rockchip-lvds.txt
@@ -25,6 +25,10 @@ Required properties:
 
 - rockchip,grf: phandle to the general register files syscon
 
+Optional properties:
+- phys : phandle for the PHY device
+- phy-names : should be "phy"
+
 Required nodes:
 
 The lvds has two video ports as described by
-- 
2.35.3

